docs/kratos/manage-identities/25_import-user-accounts-identities.mdx

```diff
@@ -574,10 +575,23 @@ The following steps are necessary to set up password migration using a web hook:
                     in: header # alternatively "cookie"
     ```
 
+    The `body` configuration field is a JSONNet snippet that will be used to map the request body to the web hook. The available
+    context is the
+    [same as for other webhooks](https://www.ory.sh/docs/guides/integrate-with-ory-cloud-through-webhooks#customizing-request-body-with-jsonnet),
+    plus these fields:
+
+    ```json
+    {
+      "identifier": "user@example.com",
+      "password": "123456"
```

Comment:
Make it explicit you have access to identity here?

Reply:
I now changed it to

    ```json5
    {
      identifier: "user@example.com",
      password: "123456",
      // also available: "identity", "flow", "request_url", "request_headers", "request_cookies", "request_method"
    }
    ```